<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Downloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QPushButton" name="PB01">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 74, 135);</string>
   </property>
   <property name="text">
    <string>Download Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB02">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 74, 135);</string>
   </property>
   <property name="text">
    <string>Download Audio</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="EdtMem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PBClr">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>80</y>
     <width>380</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 98, 112);</string>
   </property>
   <property name="text">
    <string>Clear Info</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdtURL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 215, 207);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="RBStd">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>193</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RBHQ">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>194</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>High Quality</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
